.github/workflows/build.yml

```
@@ -65,6 +65,10 @@ jobs:
           name: packages-${{ matrix.os_name }}
           path: ./artifacts/packages
 
+      - name: Publish pre-release NuGet packages to Github packages
+        run: dotnet nuget push "artifacts\packages\*.nupkg" --api-key ${{ secrets.GITHUB_TOKEN }} --skip-duplicate  --source https://nuget.pkg.github.com/justeat/index.json
+        if: ${{ runner.os == 'Windows' }}
```

Otherwise forks will start publishing packages as well, which the arbitrary forker probably doesn't want.

```suggestion
        if: ${{ github.repository_owner == 'justeat' && runner.os == 'Windows' }}
```

---

Also you'll need to not do this on PRs, fork PRs don't have a token.

---

Ahh that explains why the build isn't working - I was curious how that worked for PR's. Thanks!